raspberry-pi/4/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, pkgs, ...}:
+
+{
+  boot = {
+    kernelPackages = lib.mkDefault pkgs.linuxPackages_rpi4;
+    initrd.availableKernelModules = [ "usbhid" "usb_storage" "vc4" ];
+
+    # Needed for 4K displays
+    kernelParams = [ "cma=128M" ];
```

Comment:
Is this RAM reserved for graphic card?

Reply:
To be honest, I don't know, but searching through google most tutorials recommend it for a good default experience.

Reply:
RemoveD!